### PR TITLE
Change job running instructions

### PR DESCRIPTION
**Pull request by vojtechjelinek**
As we came up with new backup server admins policies we are also updating the wiki to reflect those policies.

### CODE REVIEW

File: Testing-jobs-and-other-features-on-production.md [hunk 1]
```diff
@@ -6,9 +6,9 @@ If your job or feature is not essential for the release and has not been fully t
 - The job or feature should run without failures on the Oppia backup server.
 - The job or feature produces the expected output.
 - The job or feature has the expected outcome (this must be verified by e.g. user-facing changes, or introspection of the datastore, or a validation job, or an output check, etc.).
-- The job or feature should be explicitly approved by server jobs admin (currently @seanlip and @vojtechjelinek).
+- The job or feature should be explicitly approved by server jobs admin (currently @seanlip,  @vojtechjelinek, @kevintab95 and @DubeySandeep).
 
-Also, in case your job is a new migration job (not any already existing migrations), there has to be an audit job accompanying it to verify that the data that you are migrating is valid in the server. **The audit job will have to be "Fully tested" before testing on the migration job can start.** 
+Also, in case your job is a new migration job (not any already existing migrations), there has to be an audit job accompanying it to verify that the data that you are migrating are valid in the server. **The audit job will have to be "Fully tested" before testing on the migration job can start.**
```
Comment (seanlip) [inline Testing-jobs-and-other-features-on-production.md]: Keep it as "is" -- data is actually plural. (The singular is datum.)

Also change "any" to "an" in the parenthetical.

File: Testing-jobs-and-other-features-on-production.md [hunk 2]
```diff
@@ -20,28 +20,32 @@ All jobs must adhere to the following requirements before they can be run on the
...
+> **Note**
+> Please submit just a single request if multiple jobs need to be run in succession. 
 
-Before submitting a testing request, please ensure that the job has already been tested manually on your local machine, and that all CI checks are passing in your PR on GitHub. For an existing job, make sure that you test the job locally and specify clearly, in the form, why you want to run the job.
+The form responses go to the server jobs admins and the admins should respond to the contributor within 24 hours. The server jobs admin should send updates after running the job(s) to the job author and [oppia release team](oppia-release-team@googlegroups.com). If the job(s) runs successfully, the admin should approve the job(s)  in the corresponding [spreadsheet row](https://docs.google.com/spreadsheets/d/1Wegd0rZhVOm3Q3VCIw0xMbLC7IWtRyrEahiPn61Fhoo/edit#gid=948463314&range=S:S) as well as merge the PR which introduces the job.
```
Comment (seanlip) [inline Testing-jobs-and-other-features-on-production.md]: should send --> will send
should approve --> will approve
Delete extra space between "job(s)" and "in".

**Review by seanlip — CHANGES_REQUESTED** [review]
Thanks! Took a pass.

**Comment by seanlip** [comment]
@vojtechjelinek please remember to use the assignees field, not the reviewers field. Thanks!

**Comment by vojtechjelinek** [comment]
~~@seanlip Yeah, I cannot change the assignees on this repo.~~

Hmm, I can edit the assignees now for some reason.

**Review by kevintab95 — APPROVED** [review]
Thanks for doing this Vojta. LGTM!

**Review by DubeySandeep — APPROVED** [review]

**Review by seanlip — APPROVED** [review]
Thanks @vojtechjelinek !

**Pull request merged**